modules/rosa-create-an-identity-based-policy.adoc

```diff
@@ -0,0 +1,169 @@
+// Module included in the following assemblies:
+//
+// * rosa-adding-additional-constraints-for-ip-based-aws-role-assumption/rosa-create-an-identity-based-policy.adoc
+
+:_content-type: PROCEDURE
+[id="rosa-create-an-identity-based-policy_{context}"]
+= Create an Identity-based IAM policy
+
+:context: rosa-create-an-identity-based-policy
+
+toc::[]
+
+You can create an Identity-based Identity and Access Management (IAM) policy that denies access to all AWS actions when the request originates from an IP address other than Red Hat provided IPs.
+
+.Prerequisites
+
+* You have access to the AWS Management Console with the permissions required to create and modify IAM policies.
+
+.Procedure
+
+. Sign in to the AWS Management Console using your AWS account credentials.
+. Navigate to the IAM service.
+. In the IAM console, select *Policies* from the left navigation menu.
+. Click *Create policy* to begin creating a new policy.
+. In the Create policy page, select the *JSON* tab to define the policy using JSON format.
```

Comment:
```suggestion
. Select the *JSON* tab to define the policy using JSON format.
```
(If you want to keep the bit about the pages, I suggest changing "In" to "On" throughout the doc).

Reply:
Updated as, "Select the *JSON* tab to define the policy using JSON format."